app/networks

```diff
@@ -1 +1 @@
-../builds/testnets
\ No newline at end of file
+/Users/fabo/Development/voyager/builds/testnets
```

Comment:
Double checking if `/Users/fabo/` is ok or not

Reply:
XD thx